extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -514,6 +514,7 @@ export class AmpSlideScroll extends BaseSlides {
    */
   showSlide_(newIndex) {
     const noOfSlides_ = this.noOfSlides_;
+    newIndex = dev().assertNumber(newIndex);
```

Comment:
To clarify: `dev.assertNumber` is stripped in PROD binary. Is this what you want?